pkg/Microsoft.ML.DnnImageFeaturizer.AlexNet/Microsoft.ML.DnnImageFeaturizer.AlexNet.nupkgproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk" DefaultTargets="Pack">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <PackageDescription>ML.NET component for pretrained AlexNet image featurization</PackageDescription>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\Microsoft.ML.OnnxTransform\Microsoft.ML.OnnxTransform.nupkgproj" />
+  </ItemGroup>
+
+  <ItemGroup>  
+    <Content Include="$(RepoRoot)\bin\obj\DnnImageModels\AlexNetOnnx\AlexNet.onnx" Pack="true" PackagePath="contentFiles\any\any\DnnImageModels\AlexNetOnnx;content\DnnImageModels\AlexNetOnnx" /> 
```

Comment:
Don't use `bin\obj` paths.

Instead, these files should be placed to `$(PackageAssetsPath)$(MSBuildProjectName)` and then you won't need these lines at all. If the assets are in the right folder, the infrastructure will just pick them up by default.